megamek/src/megamek/common/Compute.java

```diff
@@ -548,6 +548,42 @@ public static boolean isPilotingSkillNeeded(IGame game, int entityId,
                 && !isPavementStep) {
             return true;
         }
+        
+        // Sheer Cliffs, TO p.39
+        // Roads over cliffs cancel the cliff effects for units that move on roads
+        boolean quadveeVehMode = entity instanceof QuadVee 
```

Comment:
Seems to me like this logic and the logic in SharedUtility.NagPSR should be unified somehow to avoid inconsistencies if/ when it needs to be updated. Keeping it in a distinct method will also allow it to be referenced by other code that may need it (e.g. the bot)